traffic_ops/v3-client/serviceCategory.go

```diff
@@ -51,20 +51,22 @@ func (to *Session) CreateServiceCategory(serviceCategory tc.ServiceCategory) (tc
 }
 
 // UpdateServiceCategoryByName updates a service category by its unique name.
-func (to *Session) UpdateServiceCategoryByName(name string, serviceCategory tc.ServiceCategory) (tc.Alerts, ReqInf, error) {
-
+func (to *Session) UpdateServiceCategoryByName(name string, serviceCategory tc.ServiceCategory, header http.Header) (tc.Alerts, ReqInf, error) {
```

Comment:
Is changing the function signature acceptable? I know we had this discussion a while back about the consistency of function signatures and structs, for external clients' use cases not breaking.

Reply:
I might have been away when that discussion took place, but it has been discussed in the past before with most devs seemingly okay with changing the client method signatures without a deprecation period.

That said, this method is being introduced in the 5.0.0 release, which means we can still change the signature without breaking any clients (assuming this PR gets accepted into 5.0.0 which it should).